public/favicon.svg

```diff
@@ -0,0 +1 @@
+<svg width="28" height="32" viewBox="0 0 28 32" fill="currentColor" xmlns="http://www.w3.org/2000/svg"><g clip-path="url(#a)"><path d="M13.85 9.3a5.4 5.4 0 0 1-1 4.58 6.25 6.25 0 0 1-3.4 2V29.5c0 .66-.21 1.3-.68 1.77-.47.47-1.1.73-1.83.73-.6 0-1.24-.26-1.7-.73a2.56 2.56 0 0 1-.8-1.77V15.88a5.79 5.79 0 0 1-3.3-1.98 5.3 5.3 0 0 1-1-4.6L2.04.78A1 1 0 0 1 4 1v9h1.1L6.16.74c.03-.2.13-.39.28-.53a.8.8 0 0 1 .5-.2.9.9 0 0 1 .62.2c.15.14.2.33.28.53L8.9 10h1.03V1c0-.25.15-.48.25-.67.22-.18.45-.3.7-.33.23-.03.43.04.68.18.2.15.34.36.4.6l1.88 8.51ZM28 29.5a2.5 2.5 0 1 1-5 0V22h-1.05a4.06 4.06 0 0 1-3.76-2.49 5.49 5.49 0 0 1-.24-1.54v-7.1c.05-2.31.74-4.57 2.04-6.48A11.53 11.53 0 0 1 25.26.15a2 2 0 0 1 2.5.92c.16.28.18.6.24.93v27.5Z" /></g><defs><clipPath id="a"><path d="M0 0h28v32H0z"/></clipPath></defs></svg>
```

Comment:
Nit: This is outside of the current test task scope, but for the real site we'd want to have ico file too and also couple of other icons https://evilmartians.com/chronicles/how-to-favicon-in-2021-six-files-that-fit-most-needs

Reply:
100%